test/run-test-suite.js

```diff
@@ -180,24 +178,26 @@ function timeboxExpression(expr, timeout, maxDepth) {
     };
 
     // register callbacks
-    expr.assign("__evaluate_entry", function() {
+    expr.assign("__evaluate_entry", function(expr, input, env) {
+        if (env.isParallelCall) return;
         depth++;
         checkRunnaway();
     });
-    expr.assign("__evaluate_exit", function() {
+    expr.assign("__evaluate_exit", function(expr, input, env) {
+        if (env.isParallelCall) return;
         depth--;
         checkRunnaway();
     });
 }
 
 /**
- * Based on the collection of datasets and the information provided as part of the testcase,
- * determine what input data to use in the case (may return undefined).
- *
- * @param {Object} datasets Object mapping dataset names to JS values
- * @param {Object} testcase Testcase data read from testcase file
- * @returns {any} The input data to use when evaluating the jsonata expression
- */
+  * Based on the collection of datasets and the information provided as part of the testcase,
+  * determine what input data to use in the case (may return undefined).
+  *
+  * @param {Object} datasets Object mapping dataset names to JS values
+  * @param {Object} testcase Testcase data read from testcase file
+  * @returns {any} The input data to use when evaluating the jsonata expression
+  */
```

Comment:
not sure that is needed

Reply:
Ah, that would be `prettier`. I'll remove all whitespace changes in this file